src/map/party.cpp

```diff
@@ -267,7 +268,7 @@ CBattleEntity* CParty::GetMemberByName(const std::string& memberName)
 
     for (auto& member : members)
     {
-        if (strcmpi(memberName.c_str(), member->getName().c_str()) == 0)
```

Comment:
would recommend leaving this be until a refactor, if memberName has a null terminator but member->getName() doesnt, they are not equal